src/components/common/error/DetailedErrorBoundary.tsx

```diff
@@ -7,7 +7,7 @@ type ErrorBoundaryProps = PropsWithChildren & { errorTitle?: string; onCloseErro
 const DetailedErrorBoundary = ({ children, errorTitle, onCloseError }: ErrorBoundaryProps) => {
   function Fallback({ error, resetErrorBoundary }: FallbackProps) {
     return (
-      <div className='my-4 relative p-4 rounded-md bg-gray-50'>
+      <div className='my-4 relative p-4 rounded-lg bg-gray-50 h-full'>
```

Comment:
Can we not have the `h-full` for now? It blocks off the entire Report Analysis table when no workflow found

e.g. http://localhost:3000/lab/library/lib.04J5M2JFE1JPYV62RYQEG99CP4/overview (in 2xl wide mode)

Reply:
sure, I will have a check now

Reply:
"h-full" seems redundant here, delete